### PR TITLE
hf2ui31b replace delete buttons with delete icons

### PR DESCRIPTION
**Pull request by ConstantinFugaru**
Replaced delete button with delete icons for meetings and topics

### CODE REVIEW

File: lib/widgets/topic_card.dart [hunk 1]
```diff
@@ -23,36 +23,51 @@ class TopicCard extends StatelessWidget {
...
         child: Column(
           children: [
-            Padding(
-              padding: const EdgeInsets.symmetric(vertical: 5, horizontal: 0),
-              child: Text(
-                topic.content ?? "",
-                textAlign: TextAlign.left,
-                style: Theme.of(context).textTheme.subtitle2,
-              ),
+            Stack(
```
Comment (cezarmocanu) [inline lib/widgets/topic_card.dart]: I appreciate the approach, and it gets the job done, but it has some downsides:
- stacks are meant to render elements in a multi layer fashion (layering from back to front)
- the code here is a bit too bulky for only displaying a title and an icon
- items are not aligned correctly on the vertical axis
- padding is not equally applied around the items

Comment (cezarmocanu) [reply lib/widgets/topic_card.dart]: Try the following approach for solving a problem like this

```dart
Padding(
  padding: const EdgeInsets.all(8.0),
  child: Row(
    //this ensures the items from the row are aligned correctly
   //this is equivalent to using display:flex, justify-content: space-between in CSS
    mainAxisAlignment: MainAxisAlignment.spaceBetween,
    children: [
      //this needs to have the same width as the ink well
      const SizedBox(width: 24),
      Text(
        meeting.title ?? "",
        style: Theme.of(context).textTheme.subtitle2,
      ),
      Material(
        color: Colors.red,
        borderRadius: BorderRadius.circular(24),
        child: InkWell(
          borderRadius: BorderRadius.circular(24),
          child: const Padding(
            padding: EdgeInsets.all(6.0),
            child: Icon(
              Icons.delete,
              color: Colors.white,
              size: 18,
            ),
          ),
          onTap: () {
            //...
          },
        ),
      ),
    ],
  ),
),
const Divider(height: 1),
```

**Review by cezarmocanu — CHANGES_REQUESTED** [review]

File: lib/widgets/topic_card.dart (hunk 1 above)
Comment (cezarmocanu) [reply lib/widgets/topic_card.dart]: Following the previous code yields this result:
Screenshot 2022-03-01 at 00 37 23

As opposed to:
Screenshot 2022-03-01 at 00 38 55

**Comment by TeodorBirghisan** [comment]
Right now, you will merge this into main. 
**Please change the merging branch into dev.**
**In the feature, create all your branches from the dev branch.**
Changes will be merged in main only when they are ready to be deployed.

**Review by cezarmocanu — APPROVED** [review]

**Pull request merged**